components/BottomNavigation/tests/snapshot/MDCBottomNavigationBarSnapshotTests.m

```diff
@@ -270,16 +269,14 @@ - (void)testTitlePositionAdjustmentJustifiedAdjacentRegularRTL {
 
 - (void)testMaterialBaselineTheme {
   // Given
-  MDCSemanticColorScheme *colorScheme =
+  MDCContainerScheme *containerScheme = [[MDCContainerScheme alloc] init];
```

Comment:
Tip: I'd prefer if the default container scheme values were kept.  This doesn't appear to be the baseline theme if we customize the color/typography.

Reply:
Done.